design-documents/graph-ql/coverage/ProductPrices.graphqls

```diff
@@ -0,0 +1,69 @@
+interface ProductInterface{
+    ...
+    price: ProductPrices @deprecated (use price_range)
+    price_range: PriceRange!
+    tier_price: Float @deprecated (use tier_prices)
+    tier_prices: [ProductTierPrices]
+    ...
+}
+
+type ProductPrices { @deprecated (use PriceRange)
+    minimalPrice: @deprecated
+    maximalPrice: @deprecated
+    regularPrice: @deprecated
+}
+
+type ProductTierPrices {
+    customer_group_id: String @deprecated (not relevant on storefront)
+    qty: Float
+    value: Float @deprecated (use final_price)
+    final_price: Money
```

Comment:
final prices should be consistent across the schema.

Reply:
Meaning fields labeled `final_price` should all have the same type?